DASHBOARD_SETUP.md

```diff
@@ -0,0 +1,396 @@
+# Dashboard Setup Guide
+
+This guide provides comprehensive instructions for setting up dashboards with the Grafana Incremental Trace Viewer panel, including minimum width requirements and optimal configuration.
+
+## Minimum Panel Requirements
+
+The Trace Viewer panel has specific minimum size requirements to ensure optimal functionality and user experience:
+
+### Minimum Dimensions
+
+- **Width**: 600 pixels
+- **Height**: 300 pixels
+
+### Why These Requirements Matter
+
+The panel enforces these minimum dimensions because:
+
+1. **Timeline Visualization**: The trace timeline requires sufficient horizontal space to display meaningful time ranges and span durations
+2. **Span Details Panel**: The overlay drawer for span details needs adequate width to display comprehensive span information
+3. **User Interface Elements**: Buttons, controls, and text elements need proper spacing to remain accessible
+4. **Responsive Design**: The panel adapts its layout based on available space, and smaller panels would result in poor usability
+
+### Panel Size Validation
+
+The panel automatically validates its size and displays a warning if the minimum requirements are not met:
+
+```typescript
+if (width < 600 || height < 300) {
+  return (
+    <div className="flex items-center justify-center h-full p-4 text-center">
+      <div className="text-red-500">
+        <h3 className="text-lg font-semibold mb-2">⚠️ Panel too small for trace visualization</h3>
+        <p>This panel requires a minimum size of 600x300 pixels.</p>
+        <p className="text-sm text-gray-500 mt-1">
+          Current size: {width}x{height} pixels
+        </p>
+      </div>
+    </div>
+  );
+}
+```
+
+## Dashboard JSON Configuration
+
+To ensure your dashboard meets the minimum width requirements, we recommend configuring your dashboard JSON with specific grid settings.
+
+### Recommended Dashboard JSON Structure
+
+```json
+{
```

Comment:
Could you provision a dashboard instead and link it here?
You would need to add the right files to `provisioning` folder.